src/index.html

```diff
@@ -7,6 +7,417 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+      href="#"
+      class="header__logo"
+      >
+        <img
+        src="./images/logo.png"
+        alt="logo__MOYO"
+        >
+      </a>
+
+      <nav class="header__nav">
```

Comment:
Make nav a separate BEM block, and not like header element, if you need to add some extra CSS props like indents use mix classes according to BEM.